frontend/src/app/services/statistics/statistics.service.ts

```diff
@@ -12,7 +12,7 @@ import { BehaviorSubject, Observable } from 'rxjs';
 })
 export class StatisticsService {
 
-  private _statisticsnKey = 'questions';
```

Comment:
😬

Reply:
jsem si říkala, že to bude asi kod ve 3 ráno :D